src/universal/components/RetroGroupPhase.js

```diff
@@ -50,7 +53,7 @@ const RetroGroupPhase = (props: Props) => {
           iconLarge
           iconPalette="warm"
           iconPlacement="right"
-          label={'Done! Let’s Vote'}
+          label={`Done! Let’s ${nextPhaseLabel}`}
```

Comment:
kinda crazy that each phase of the retro has a label that is both a noun & a verb...

Reply:
Yeah, I think it’s kind of playful in a simple way.